spec2vec/model_building.py

```diff
@@ -0,0 +1,129 @@
+from typing import List
+from typing import Tuple
+from typing import Union
+import gensim
+from spec2vec.utils import ModelSaver
+from spec2vec.utils import TrainingProgressLogger
+
+
+def train_new_word2vec_model(documents: List, iterations: Union[List, int], filename: str = None, **kwargs):
+    """Train a new Word2Vec model (using gensim). Save to file if filename is given.
+
```

Comment:
A example snippet how to use it would help me.

Reply:
I added a snipped code example to the docstring.

Reply:
nice thanks